src/citrine/resources/design_space.py

```diff
@@ -69,7 +69,8 @@ def create_default(self,
                        predictor_id: UUID,
                        include_ingredient_fraction_constraints: bool = False,
                        include_label_fraction_constraints: bool = False,
-                       include_label_count_constraints: bool = False) -> DesignSpace:
+                       include_label_count_constraints: bool = False,
+                       include_parameter_constraints: bool = False) -> DesignSpace:
```

Comment:
Why are we using `False` here instead of `None`?  Seems like a painful way to make later migration annoying.